docs/collecting-data/collecting-from-own-applications/mobile-trackers/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Mobile Native Trackers"
+title: "Mobile native Trackers"
```

Comment:
```suggestion
title: "Mobile native trackers"
```

Reply:
I left this one in because it looked weird to have all the other trackers as Trackers down the side. I'd rather do them all in one go

Reply:
It does look a bit weird to have the mixed case (also with the "React Native Tracker" under it). I'll change back to "Mobile Native Trackers" and we can change to sentence case in one go as Miranda says.